main.py

```diff
@@ -0,0 +1,20 @@
+import streamlit as st
+
+from PIL import Image
+import requests
+from backend import model
+
+def __main():
+    st.write("Choose an image file with a line of text:")
+    image_file = st.file_uploader("Choose an image")
+    st.write("To start, please specify whether the text is printed or handwritten.")
+    printed = st.button("printed")
+    handwritten = st.button("handwritten")
```

Comment:
I think [radio button](https://docs.streamlit.io/library/api-reference/widgets/st.radio) fits better here

Reply:
fixed +replaced models with smaller ones, they seem to work just as fine but faster